Documentation/GeneralConventions/GuidesXml.rst

```diff
@@ -0,0 +1,197 @@
+..  _guides-xml:
+
+================================
+:file:`Documentation/guides.xml`
+================================
+
+This XML file contains meta information and configuration used during rendering
```

Comment:
In general I suggest to start with a full guides.xml example, so people can see what the file looks like without reading the whole chapter.